components.yaml

```diff
@@ -41,6 +41,6 @@ MAPL:
 GEOSgcm_GridComp:
   local: ./src/Components/GEOSldas_GridComp/@GEOSgcm_GridComp
   remote: ../GEOSgcm_GridComp.git
-  branch: develop
+  branch: feature/rreichle/snow_check_after_apply_LDAS_incr
```

Comment:
@biljanaorescanin: Make sure to restore components.yaml from the develop branch before merging this PR.